theme/lumo/vaadin-combo-box-dropdown-styles.html

```diff
@@ -84,6 +84,14 @@
           transform: rotate(360deg);
         }
       }
+
+      /* RTL specific styles */
+
+      :host([loading]) [part~="loader"] {
+        left: auto;
+        margin-left: 0;
+        margin-right: auto;
```

Comment:
This probably needs to also override `margin-inline-start` and `margin-inline-end`.

Reply:
Good point. Weird that we have used those as they are not supported by ie11. Updated.